oauth2_provider/migrations/0003_auto_20201211_1314.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.4 on 2020-12-11 13:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('oauth2_provider', '0002_auto_20190406_1805'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='grant',
```

Comment:
I need to read about swappable models and migrations, not sure how the magic works for migrations

Reply:
That is a good point. I didn't realize it.

I found very similar example in Django code. Swappable `User` model:

In [contrib/auth/migrations/0001_initial.py](https://github.com/django/django/blob/master/django/contrib/auth/migrations/0001_initial.py) is initialized User model as swappable.

In [contrib/auth/migrations/0003_alter_user_email_max_length.py](https://github.com/django/django/blob/master/django/contrib/auth/migrations/0003_alter_user_email_max_length.py) is updated `max_length` for `email` field and looks like normal migration.

Reply:
cool!